.solhintignore

```diff
@@ -0,0 +1 @@
+contracts/hardhat-dependency-compiler
```

Comment:
Added to get tests to pass solhint error:  
contracts/hardhat-dependency-compiler/@pooltogether/pooltogether-proxy-factory/contracts/GenericProxyFactory.sol
  2:1  error  Compiler version >0.0.0 does not satisfy the ^0.6.0 semver requirement  compiler-version
  3:8  error  Use double quotes for string literals